src/components/deploy/deploy_args.rs

```diff
@@ -63,24 +64,41 @@ impl From<SignerContext> for near_cli_rs::commands::ActionContext {
                     return Ok(prepopulated_transaction);
                 }
 
-                let call_result = network_config
-                    .json_rpc_client()
-                    .blocking_call_view_function(
-                        near_social_account_id,
-                        "get",
-                        serde_json::to_string(&crate::socialdb_types::SocialDbQuery {
-                            keys: local_widgets
-                            .keys()
+                let runtime = tokio::runtime::Builder::new_multi_thread().enable_all().build()?;
+                let chunk_size = 10;
+                let mut calls = Vec::with_capacity(local_widgets.len() / chunk_size + 1);
+                let local_widget_names = local_widgets.keys().cloned().collect::<Vec<_>>();
+
+                for (index, _) in local_widget_names.iter().enumerate().step_by(chunk_size) {
+                    let args = serde_json::to_string(&crate::socialdb_types::SocialDbQuery {
+                        keys: local_widget_names.iter().skip(index).take(chunk_size)
```

Comment:
There is a nice [`chunks` method available on slices in the Rust standard library](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.chunks). Please, use it, it will significantly simplify the code.

Reply:
I new about the `array_chunks` but not `chunks` method. I will refactor it.